### PR TITLE
Add address field to geofence

### PR DESCRIPTION
**Pull request by bschoeneweis**
## What?
Adding details on the `address` field to the CSV import section for geofences and geofence upsert API endpoint.

### CSV import fields
https://documentation-2ak1igamf-radarlabs.vercel.app/documentation/geofences#csv-import
image

### Example CSV
https://documentation-2ak1igamf-radarlabs.vercel.app/documentation/geofences#csv-import
Screen Shot 2022-04-27 at 12 43 54 PM

### Upsert API spec
https://documentation-2ak1igamf-radarlabs.vercel.app/documentation/api#upsert-a-geofence
image

### CODE REVIEW

File: docs/api.mdx [hunk 1]
```diff
@@ -1589,7 +1589,8 @@ Upserts a geofence. The geofence can be uniquely referenced by `tag` and `extern
...
+- **`coordinates`** (array, required if `address` is not included): An array or JSON string representing a center for type `circle` or a destination for type `isochrone` in the format `[longitude,latitude]`. A two-dimensional array or JSON string representing a closed ring of between 4 and 2,000 coordinates in the format `[[longitude0, latitude0],[longitude1,latitude1],[longitude2,latitude2],...,[longitude0,latitude0]]` for type `polygon`. **Note that longitude comes before latitude, a GeoJSON convention.**
+- **`address`** (string, required if `coordinates` are not included): An address to search for, and if found, will represent the center for type `circle` or destination for type `isochrone`. If `address` and `coordinates` are both provided, they must be within a certain distance of each other, and `coordinates` will take precedent. Ignored for type `polygon`.
```
Comment (jsani-radar) [inline docs/api.mdx:1593]: "will represent the center for type `circle` or `isochrone`" - don't think we need to mention destination

Comment (jsani-radar) [inline docs/api.mdx:1593]: "If `address` and `coordinates` are both provided, they must be nearby..."

File: docs/geofences.mdx [hunk 2]
```diff
@@ -89,14 +90,15 @@ The CSV may have the following columns:
...
+"SoHo",neighborhood,soho,polygon,,,"[[-73.996973,40.725660],[-73.998282,40.726294],[-74.001072,40.727660],[-74.002853,40.728603],[-74.003282,40.727936],[-74.003840,40.726310],[-74.004226,40.725351],[-74.004998,40.723530],[-74.005191,40.722993],[-74.005556,40.722115],[-74.004462,40.721334],[-74.001908,40.719496],[-73.999827,40.718016],[-73.997467,40.720863],[-73.997016,40.722212],[-73.996501,40.723497],[-73.995750,40.724278],[-73.995128,40.725107],[-73.996973,40.725660]]",,true,,,,,
+"15 minutes driving to Disney World","car-15",disney,isochrone,15,car,"[-81.563874,28.385233]",,true,"{""type"":""resort""}",,,,
+"Stopped Empire State Building",stopped,456,circle,50,,"[-73.9878531,40.7484445]",,true,,true,,,
+"Madison Square Garden",address,123,circle,50,,,"4 Pennsylvania Plaza, New York, NY 10001",true,,,,,
+"Disable and delete after New Years",newyears,2027,circle,50,,"[-73.9877313,40.7579787]",,true,,,"2027-01-01T00:01:00-05:00","Jan 2, 2027",
+"Disable and delete after New Years 2",newyears2,2027,circle,50,,"[-73.9887289,40.7561842]",,true,,,1/1/27,2027/1/2,
+"Simulated user geofence",user,simulated,circle,50,,"[-73.9934387,40.7505045]",,true,,,,,"simulated-user-id"
```
Comment (jsani-radar) [inline docs/geofences.mdx:101]: small nit - do you mind sharing a successful screenshot of the import flow with this new file?

**Review by jsani-radar — COMMENTED** [review]
few small comments, overall looks good and will be 👍🏽 on next pass

just a heads up we'll need to update the import modal on the FE to have the same file snippet

File: docs/geofences.mdx (hunk 2 above)
Comment (bschoeneweis) [reply docs/geofences.mdx:101]: You got it!  Just ran this again.

image

image

File: docs/api.mdx (hunk 1 above)
Comment (bschoeneweis) [reply docs/api.mdx:1593]: Nice yeah I like this better

Comment (bschoeneweis) [reply docs/api.mdx:1593]: 👍

File: docs/geofences.mdx (hunk 2 above)
Comment (jsani-radar) [reply docs/geofences.mdx:101]: bless thank you 🙏🏽 just wanted to confirm things are still working e2e

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**